test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -165,6 +182,52 @@ func describeQatDpdkPlugin() {
 			ginkgo.It("does nothing", func() {})
 		})
 	})
+
+	ginkgo.Context("When QAT's autoreset is off [Functionality:heartbeat]", func() {
+		ginkgo.BeforeEach(func() {
+			ginkgo.By("creating a configMap before plugin gets deployed")
+			e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "create", "configmap", "--from-literal", "qat.conf=AutoresetEnabled=off", "qat-config")
```

Comment:
Maybe I don't understand the test flow completely, but maybe the resource type could also be set here so we don't have to guess/detect it?

Reply:
Yeah, that's one of the things that I was thinking. But, then I have no idea if it is possible to put two things with `--from-literal`.
Or, do we make a file `qat.conf` and put it with `--from-file`?

Reply:
At least this seems to work:
```
# kubectl create configmap --dry-run -o yaml testing --from-literal 'foobar=true' --from-literal 'bar=foo'
apiVersion: v1
data:
  bar: foo
  foobar: "true"
kind: ConfigMap
metadata:
  creationTimestamp: null
  name: testing
```

Reply:
Well, I know you can put two `from-literal` but what about `qat.conf` file name? I could not find out a way to do. If there is a way, please let me know.!

Reply:
> $ kubectl describe cm qat-config
Name:         qat-config
Namespace:    default
Labels:       <none>
Annotations:  <none>

Data
====
qat.conf:
----
AutoresetEnabled=off

BinaryData
====

Events:  <none>

Reply:
I found a way though it is different to what you suggetsed.! Still thank you for letting me to think about this matter again. I thought it is not possible.